src/Slices/Home/UI/EnvironmentsOverview.test.tsx

```diff
@@ -49,7 +50,9 @@ test("Given environments overview When filtering by project Then only the matchi
     name: "Environment card",
   });
   expect(initialCards).toHaveLength(4);
-  const input = await screen.findByPlaceholderText("Filter by project");
+  const input = await screen.findByPlaceholderText(
+    words("home.filters.project.placeholder")
+  );
   await userEvent.click(input);
   const option = await screen.findByRole("option", { name: "default" });
```

Comment:
if default is printed to the screen, this has to be extracted too.

Reply:
There is no such a text visible in home screen